easybuild/easyconfigs/p/PyTorch/PyTorch-1.7.1-foss-2020b.eb

```diff
@@ -0,0 +1,310 @@
+        'download_filename': '072586a71b55b7f8c584153d223e95687148a900.tar.gz',
+        'filename': 'psimd-20200517.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'psimd',
+    },
+    {
+        'source_urls': ['https://github.com/Maratyszcza/pthreadpool/archive'],
+        'download_filename': '029c88620802e1361ccf41d1970bd5b07fd6b7bb.tar.gz',
+        'filename': 'pthreadpool-20200615.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'pthreadpool',
+    },
+    {
+        'source_urls': ['https://github.com/Maratyszcza/PeachPy/archive'],
+        'download_filename': '07d8fde8ac45d7705129475c0f94ed8925b93473.tar.gz',
+        'filename': 'PeachPy-20180218.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'python-peachpy',
+    },
+    {
+        'source_urls': ['https://github.com/shibatch/sleef/archive'],
+        'download_filename': '7f523de651585fe25cade462efccca647dcc8d02.tar.gz',
+        'filename': 'sleef-20190730.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'sleef',
+    },
+    {
+        'source_urls': ['https://github.com/01org/tbb/archive'],
+        'download_filename': 'a51a90bc609bb73db8ea13841b5cf7aa4344d4a9.tar.gz',
+        'filename': 'tbb-20181009.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'tbb',
+    },
+    {
+        'source_urls': ['https://github.com/pytorch/tensorpipe/archive'],
+        'download_filename': '95ff9319161fcdb3c674d2bb63fac3e94095b343.tar.gz',
+        'filename': 'tensorpipe-20200928.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'tensorpipe',
+    },
+    {
+        'source_urls': ['https://github.com/google/googletest/archive'],
+        'download_filename': '2fe3bd994b3189899d93f1d5a881e725e046fdc2.tar.gz',
+        'filename': 'googletest-20180831.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'tensorpipe/third_party/googletest',
+    },
+    {
+        'source_urls': ['https://github.com/google/libnop/archive'],
+        'download_filename': 'aa95422ea8c409e3f078d2ee7708a5f59a8b9fa2.tar.gz',
+        'filename': 'libnop-20200723.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'tensorpipe/third_party/libnop',
+    },
+    {
+        'source_urls': ['https://github.com/libuv/libuv/archive'],
+        'download_filename': '02a9e1be252b623ee032a3137c0b0c94afbe6809.tar.gz',
+        'filename': 'libuv-20200419.tar.gz',
+        'extract_cmd': local_extract_cmd_pattern % 'tensorpipe/third_party/libuv',
+    },
+]
+patches = [
+    'PyTorch-1.6.0_fix-test-dataloader-fixed-affinity.patch',
+    'PyTorch-1.7.0_fix_altivec_defines.patch',
+    'PyTorch-1.7.0_fix_test_DistributedDataParallel.patch',
+    'PyTorch-1.7.0_fix-fbgemm-not-implemented-issue.patch',
+    'PyTorch-1.7.0_avoid-nan-in-test-torch.patch',
+    'PyTorch-1.7.0_increase-distributed-test-timeout.patch',
+    'PyTorch-1.7.0_disable-dev-shm-test.patch',
+    'PyTorch-1.7.1_el8_ppc64le.patch',
```

Comment:
This patch is not needed for a `foss` build. (It is also not needed when we build with CUDA 11).